### ISSUE TITLE
Select a list using checkboxes

### ISSUE THREAD

**Issue by nmk** [etaque/elm-form#80]
I am trying to figure out how to select a list of values using checkboxes. I have a (non-working) gist of what I am trying to achieve here: https://gist.github.com/fc0e1594438e336aa80f13e91a8fe262

Any help would be appreciated!

**Comment by etaque** [comment]
Hello,
It's easier now that we have `sequence` in validations (I just published 2.0.0).
I forked your gist, see `checkboxList` function and usage:
https://gist.github.com/etaque/9f8d50a4a9de0a105adcfaff0426d410#file-main-elm-L39-L55

This function should probably make his way into the package, but I'm not sure of the name: we already have `list` and `sequence`, things are getting confusing here.

**Comment by etaque** [comment]
(cc @amitaibu re: `sequence` usage)

**Comment by nmk** [comment]
Ah, `sequence` in the new version is great! Thank you for looking into this. I agree that something like `checkboxList` should be a part of `Form.Validate`. I found a minor inconvenience though — if the keys are numeric, the selection does not work. I have updated the gist, have a look at the difference between `colors` and `colorsWithNumericKeys`.

This can be worked around by prefixing the field name with some constant string and adjusting the options, but it took me a while to figure out why it wasn't working.

I suspect numeric keys would be the norm if the things being selected over come from the backend.
